main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Dialogs/NewConfigurationDialog.cs

```diff
@@ -0,0 +1,190 @@
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+using System;
+using System.Collections.Generic;
+using MonoDevelop.Projects;
+using MonoDevelop.Core;
+using Xwt;
+using MonoDevelop.Components;
+
+namespace MonoDevelop.Ide.Gui.Dialogs
+{
+	class NewConfigurationDialog : Xwt.Dialog
+	{
+		public string ConfigName {
+			get {
+				string plat = MultiConfigItemOptionsPanel.GetPlatformId (comboPlatform.TextEntry.Text.Trim ());
+				if (string.IsNullOrEmpty (plat))
+					return comboName.TextEntry.Text.Trim ();
+				else
+					return comboName.TextEntry.Text.Trim () + "|" + plat;
+			}
+			set {
+				int i = value.LastIndexOf ('|');
+				if (i == -1) {
+					comboName.TextEntry.Text = value;
+					comboPlatform.TextEntry.Text = string.Empty;
+				} else {
+					comboName.TextEntry.Text = value.Substring (0, i);
+					comboPlatform.TextEntry.Text = MultiConfigItemOptionsPanel.GetPlatformName (value.Substring (i + 1));
+				}
+			}
+		}
+
+		public bool CreateChildren {
+			get => createChildrenCheck.Active;
+		}
+
+		public NewConfigurationDialog (ItemConfigurationCollection<ItemConfiguration> configurations)
+			: this (null, configurations)
+		{
+		}
+
+		public NewConfigurationDialog (IConfigurationTarget item, ItemConfigurationCollection<ItemConfiguration> configurations)
+		{
+			Build ();
+
+			this.configurations = configurations;
+
+			SetupConfigs (item);
+
+			comboName.SetFocus ();
+		}
+
+		void Build ()
+		{
+			Padding = 6;
+			Resizable = false;
+			Title = GettextCatalog.GetString ("New Configuration");
+
+			var mainVBox = new VBox () { Spacing = 6 };
+			var table = new Table { DefaultColumnSpacing = 6, DefaultRowSpacing = 6 };
+
+			var label = new Label { Text = GettextCatalog.GetString ("Name:") };
+			comboName = new ComboBoxEntry ();
+			comboName.TextEntry.Changed += ComboTextChanged;
+			comboName.Accessible.LabelWidget = label;
+			table.Add (label, 0, 0);
+			table.Add (comboName, 1, 0);
+
+			label = new Label { Text = GettextCatalog.GetString ("Platform:") };
+			comboPlatform = new ComboBoxEntry ();
+			comboPlatform.TextEntry.Changed += ComboTextChanged;
+			comboPlatform.Accessible.LabelWidget = label;
+			comboPlatform.WidthRequest = 250;
+			table.Add (label, 0, 1);
+			table.Add (comboPlatform, 1, 1);
+
+			popover = new InformationPopoverWidget ();
```

Comment:
move this declaration up and set the a11y label to the Name label.
```suggestion
			popover = new InformationPopoverWidget ();
			popover.Accessible.LabelWidget = label;
```